client/components/mma/delivery/records/ReadOnlyAddressDisplay.tsx

```diff
@@ -50,6 +50,7 @@ export const ReadOnlyAddressDisplay = (props: ReadOnlyAddressDisplayProps) => {
 					css={css`
 						${dtCss()}
 					`}
+					data-qm-masking="blocklist"
```

Comment:
again just masking a label here

Reply:
Also, not needed as the `DeliveryAddressDisplay` component already includes masking.